src/borrowing/examples.md

```diff
@@ -0,0 +1,33 @@
+# Borrow Errors
```

Comment:
This will need timing information, similar to other slides. Maybe 3 minutes?

Reply:
Added!